config/config.py.template

```diff
@@ -375,7 +375,7 @@ MLLIB_PERF_TEST_RUNNER = "mllib.perf.TestRunner"
 #  * Set `USE_CLUSTER_SPARK = True` and `MLLIB_SPARK_VERSION = {desired Spark version, e.g. 1.5}`
 #  * Don't use PREP_MLLIB_TESTS = True; instead manually run `cd mllib-tests; sbt/sbt -Dspark.version=1.5.0-SNAPSHOT clean assembly` to build perf tests
 
-MLLIB_SPARK_VERSION = 2.0
+MLLIB_SPARK_VERSION = 2.0.0
```

Comment:
My environment is a little customized by the time I run this code (so line numbers don't match), but I am seeing this:

Loading configuration from <snip>/spark-perf/config/config.py
Traceback (most recent call last):
  File "./bin/../lib/sparkperf/main.py", line 40, in <module>
    config = imp.load_source("config", "", cf)
  File "", line 410
    MLLIB_SPARK_VERSION = 2.0.0